### ISSUE TITLE
Next 13.5.0 is not pushed to the npm registry

### ISSUE THREAD

**Issue by hstemplewski** [vercel/next.js#55599]
### Link to the code that reproduces this issue

https://codesandbox.io/p/sandbox/summer-shape-f7992k?file=/package.json:19,1

### To Reproduce

1. `npm i next@13.5.0` or `npm i next`

### Current vs. Expected behavior

Not getting the error:
```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/next/-/next-13.5.0.tgz - Not found
npm ERR! 404
npm ERR! 404  'next@https://registry.npmjs.org/next/-/next-13.5.0.tgz' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/xxxxxxxxxxx/.npm/_logs/2023-09-19T16_41_52_628Z-debug-0.log
```

### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Not relevant
```

### Which area(s) are affected? (Select all that apply)

Package manager (npm, pnpm, Yarn)

**Comment by techies23** [comment]
I was just about to report this as well. Getting same issue.

**Comment by vitorsbmendes** [comment]
Also facing this

**Comment by periface** [comment]
Im facing this issue too 😐

**Comment by g00gol** [comment]
Getting same issue. 404 error on npm registry

**Comment by mayrsascha** [comment]
Same when you install it with yarn...

**Comment by k4rmine** [comment]
I have the same problem, I was about to publish it.

**Comment by ElectricCodeGuy** [comment]
Yep same issue xD

**Comment by azhardumac** [comment]
same

**Comment by hongz1** [comment]
same

**Comment by nolannbiron** [comment]
same issue, going back to 13.4.19

**Comment by KingBael09** [comment]
Yup same.

**Comment by tebuevd** [comment]
Same

**Comment by iwoork** [comment]
Same here!

**Comment by castrosu** [comment]
From 13.4.19-canary41 to 13.5.0 to 13.5.1-canary1, someone did a whoopsie at vercel

**Comment by adamcanray** [comment]
same here

**Comment by bhuelga** [comment]
Same here

**Comment by davecarlson** [comment]
Why are you trying to install a package that hasn't been published or released yet ?

**Comment by vitorsbmendes** [comment]
> Why are you trying to install a package that hasn't been published or released yet ?

Just do a "npx create-next-app@latest" and the error also occurs

**Comment by azhardumac** [comment]
> > Why are you trying to install a package that hasn't been published or released yet ?
> 
> Just do a "npx create-[next-app](https://linear.app/vercel/issue/NEXT-app)@latest" and the error also occurs

i cant do any versions, which ones working?

**Comment by davecarlson** [comment]
It's literally in the process of being built and released right now

**Comment by KingBael09** [comment]
> > > Why are you trying to install a package that hasn't been published or released yet ?
> > 
> > 
> > Just do a "npx create-[next-app](https://linear.app/vercel/issue/NEXT-app)@latest" and the error also occurs
> 
> i cant do any versions, which ones working?

Bad timing huh?

**Comment by adamcanray** [comment]
> It's literally in the process of being built and released right now, maybe just hold off for a few minutes :) npm especially takes time to propograte:
> 
> Even this page flip flops on reload to it showing / not showing. https://www.npmjs.com/package/next?activeTab=versions
> 
> Patience people

Patient huh... my boss is waiting right behind me.

**Comment by adamcanray** [comment]
> > It's literally in the process of being built and released right now, maybe just hold off for a few minutes :) npm especially takes time to propograte:
> > Even this page flip flops on reload to it showing / not showing. https://www.npmjs.com/package/next?activeTab=versions
> > Patience people
> 
> Patient huh... my boss is waiting right behind me.

I'm going back to 13.4.19

**Comment by joulev** [comment]
Best way is to wait until this is fixed, which shouldn’t take long. But if you need it urgently:

* Existing apps: pin Next.js dependencies (next, the eslint package, etc) to 13.4.19
* New apps: instead of using create-next-app, clone a known working repository and continue from there, with Next.js version pinned to 13.4.19. This should be one of such working boilerplates: https://vercel.com/templates/next.js/nextjs-boilerplate

**Comment by nolannbiron** [comment]
Release action is running guys

**Comment by farhan2077** [comment]
Here I thought that it was a problem from my end 😆

**Comment by ericklarsen** [comment]
same here hahaha

**Comment by danilovvvladimir** [comment]
same :)

**Comment by azhardumac** [comment]
FINALLY

**Comment by hstemplewski** [comment]
> > > Why are you trying to install a package that hasn't been published or released yet ?
> > 
> > 
> > Just do a "npx create-[next-app](https://linear.app/vercel/issue/NEXT-app)@latest" and the error also occurs
> 
> i cant do any versions, which ones working?

After you try `npx create-next-app@13.14.9` or any other you will get the same error because the step of creation is installing latest version of next ;)

**Comment by davecarlson** [comment]
> > It's literally in the process of being built and released right now, maybe just hold off for a few minutes :) npm especially takes time to propograte:
> > Even this page flip flops on reload to it showing / not showing. https://www.npmjs.com/package/next?activeTab=versions
> > Patience people
> 
> Patient huh... my boss is waiting right behind me.

Explain to your boss that it's important to not just deploy to prod on the 'latest' release, without testing it first. Concerning your package is configured to jump straight to 13.5....

**Comment by joulev** [comment]
13.5.1 is released. Should work now, everything back to normal.

**Comment by hstemplewski** [comment]
nice work ;) fixed with 13.15.1. Closing the issue.

**Comment by sandy2196** [comment]
Issue fixed. Thanks @joulev

**Comment by balazsorban44** [comment]
Thanks for opening an issue @hstemplewski.

For commenters, for future reference, "same issue" and similar comments have no value and only add noise and are considered off-topic/spam, not just in this particular issue, but any bug reports. Please use the :+1: upvote on the the issue description in the future, unless you can provide more context about the issue! If you just want to follow the issue, subscribe to notifications instead of adding a comment. Thanks! :pray:

> From 13.4.19-canary41 to 13.5.0 to 13.5.1-canary1, someone did a whoopsie at vercel

For the curious, npm had an outage and the stable release was silently failing, halting the release process, so we needed to wait until it was resolved and shipped 13.5.1 as soon as npm was up again. We are sorry for the inconvenience! Happy shipping!
